builder/comp-builder.nix

```
@@ -545,7 +545,7 @@ let
               # we assume that if the SETUP_HS command fails and the following line was found in the error
               # log, that it was the only error. Hence if we do _not_ find the line, grep will fail and this derivation
               # will be marked as failure.
-              cat $SETUP_ERR | grep 'No executables and no library found\. Nothing to do\.'
+              cat $SETUP_ERR | grep 'No executables and no library found\. Nothing to'
```

Isn't this more robust? In case we push this even further, with an even longer name?
```suggestion
              cat $SETUP_ERR | tr -d '\n\r' | grep 'No executables and no library found\. Nothing to do\.'
```